lib/hooks/usePoolChainValues.js

```diff
@@ -154,6 +193,72 @@ export const fetchPoolChainValues = async (
   }
 }
 
+const fetchSablierStreamChainValues = async (
+  provider,
+  chainId,
+  sablierManagerAddress,
+  prizePoolAddress,
+  prizeStrategyData
+) => {
+  try {
+    const sablierManagerContract = contract(
+      'sablierManager',
+      SablierManagerAbi,
+      sablierManagerAddress
+    )
+    const { sablierManager } = await batch(
+      provider,
+      sablierManagerContract.sablierStreamId(prizePoolAddress)
+    )
+    const streamId = sablierManager.sablierStreamId[0]
+
+    const sablierContract = contract('sablier', SablierAbi, CONTRACT_ADDRESSES[chainId].Sablier)
+    const sablierValues = await batch(provider, sablierContract.getStream(streamId))
+    const tokenContract = contract(
+      'sablierToken',
+      ERC20Abi,
+      sablierValues.sablier.getStream.tokenAddress
+    )
+    const tokenValues = await batch(provider, tokenContract.decimals().symbol().name())
+
+    const { amountThisPrizePeriod, amountPerPrizePeriod } = calculateSablierPrize(
+      {
+        startTime: sablierValues.sablier.getStream.startTime,
```

Comment:
More places we can improve readability by pulling batch() results into their own vars.